skimage/measure/fit.py

```diff
@@ -756,18 +758,21 @@ def ransac(data, model_class, min_samples, residual_threshold,
     >>> sum(inliers) > 40
     True
 
-    Robustly estimate geometric transformation:
+    Robustly estimate geometric transformation. The argument `min_samples`
+     is an absolute number. If instead we want to use a proportion of the full
```

Comment:
indentation is off here. 

```suggestion
    is an absolute number. If instead we want to use a proportion of the full
```

But, more critically, I don't think the new rephrasing works, because "Robustly estimate geometric transformation" works as a section title, not as a sentence. How about:

"""
RANSAC can be used to robustly estimate a geometric transformation. In this section, we also
show how to use a proportion of the total samples, rather than an absolute number.
"""